addons/stock/models/stock_move_line.py

```diff
@@ -42,10 +43,15 @@ class StockMoveLine(models.Model):
     from_loc = fields.Char(compute='_compute_location_description')
     to_loc = fields.Char(compute='_compute_location_description')
     lots_visible = fields.Boolean(compute='_compute_lots_visible')
-    state = fields.Selection(related='move_id.state')
+    state = fields.Selection(related='move_id.state', store=True)
     consume_line_ids = fields.Many2many('stock.move.line', 'stock_move_line_consume_rel', 'consume_line_id', 'produce_line_id', help="Technical link to see who consumed what. ")
     produce_line_ids = fields.Many2many('stock.move.line', 'stock_move_line_consume_rel', 'produce_line_id', 'consume_line_id', help="Technical link to see which line was produced with this. ")
 
+    @api.multi
+    def compute_uom_id(self):
+        for line in self:
+            line.uom_reserved_id = line.product_uom_id
+
```

Comment:
Same comment as for `uom_reserved_id` on `stock.quant`. Don't think we need this here.